modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -3,6 +3,14 @@
 // * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
 // * installing/installing_restricted_networks/installing-restricted-networks-aws.adoc
 
+ifeval::["{context}" == "installing-restricted-networks-aws"]
+:restricted:
+:aws:
```

Comment:
This is strange inside a `installation-generate-aws-user-infra-ignition.adoc` module.  Can we drop `:aws:` and `ifdef::aws[]` conditionals in this file?  Or rename to a generic `installation-generate-user-infra-ignition.adoc`?